__tests__/api.test.js

```diff
@@ -127,6 +127,38 @@ describe('GET /api/articles', () => {
         });
     });
 
+    test('200: responds with articles filtered by the topic query provided', () => {
+      return request(app)
+      .get('/api/articles?topic=cats')
+      .expect(200)
+      .then(({ body })=> {
+        const { articles } = body;
+        
+        expect(articles).toHaveLength(1)
+        articles.forEach((article)=> {
+          expect(article).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            topic: 'cats',
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(Number)
+          })
+        })
+      })
+    });
+
```

Comment:
Topic=paper exists but does not have any article associated with it, we should respond with a 200 and return an empty array in this case

Reply:
Thank you Micheal. I will adjust accordingly